config/zones/CH.yaml

```diff
@@ -73,11 +73,30 @@ emissionFactors:
         source: Electricity Maps, 2021 average
         value: 113.07156762843732
     unknown:
-      _comment:
-        'Source ENTSOE, Swiss government and Electricity Maps: https://github.com/electricitymaps/electricitymaps-contrib/pull/2898
-        and https://docs.google.com/spreadsheets/d/1FLHQ6e9Es08BIqX654BM3SEb_fuAk4k4O-6cmRXyw_E/'
-      source: 52% hydro storage, 48% unknown thermal
-      value: 288
+      - datetime: '2017-01-01'
```

Comment:
In the API we have data going back as far as 2015, do we have the values needed to calculate the unknown CO2eq value for 2015 and 2016 as well?

Reply:
In that case it defaults to the oldest date. Definition for how we parse them:
```
`n` dates are sorted in ascending order (d1, d2, ..., dn)
d1 is valid from from (epoch to d2)
d2 is valid from (d2 to d3)
dn is valid from (dn to end_of_time)
```